tests/RobotFramework/tests/cumulocity/configuration/configuration_operation_multi_cloud.robot

```diff
@@ -79,17 +75,13 @@ Setup Second Device
     Execute Command    tedge config set c8y@second.bridge.topic_prefix c8y-second
     Execute Command    tedge config set c8y@second.url "$(tedge config get c8y.url)"
 
-    Execute Command    tedge cert create c8y@second --device-id ${child_sn}
+    Execute Command    tedge cert create c8y@second --device-id ${second_device_sn}
     Execute Command
     ...    cmd=sudo env C8Y_USER='${C8Y_CONFIG.username}' C8Y_PASSWORD='${C8Y_CONFIG.password}' tedge cert upload c8y@second
 
     Execute Command    tedge connect c8y@second
```

Comment:
@albinsuresh pointed out doing this will bypass the automatic cleanup we get for the devices we create. I'm not sure exactly how best to perform this clean-up. @reubenmiller are you able to help with this?

Reply:
We can ignore the cleanup for now (for this PR). We can easily add a new keyword to register a device for cleanup once the suite has finished.